src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb

```diff
@@ -406,6 +404,17 @@ def _upload_page_blob_in_threads(file_path, file_size, container_name, blob_name
       threads.each(&:join)
       cloud_error(standard_errors.to_s) unless standard_errors.empty?
     end
+
+    def _create_page_blob(container_name, blob_name, file_size, options)
+      @blob_service_client.create_page_blob(container_name, blob_name, file_size, options)
+    rescue StandardError => e
+      if e.inspect.include?('ContainerNotFound')
+        @blob_service_client.create_container(container_name, {})
+        @blob_service_client.create_page_blob(container_name, blob_name, file_size, options)
+      else
+        raise e
+      end
+    end
   end
 
   private
```

Comment:
I think, the `private` in line 411 should be removed and the `private` in line 299 should be kept.

Reply:
L281 already have a private.

Reply:
I think private in L420 is for the classes.

Reply:
yes. you are right